command/ping.go

```diff
@@ -14,6 +23,28 @@ func (cmd *Command) ping() {
 }
 
 func (cmd *Command) checkQuorumStatus() {
+	if cmd.quorumAddr == "" {
```

Comment:
We should change from `period.did` to `did` if we haven't already for quorum status check